parser/parser.y

```diff
@@ -952,13 +953,15 @@ import (
 	CreateBindingStmt          "CREATE BINDING  statement"
 	CreatePolicyStmt           "CREATE PLACEMENT POLICY statement"
 	CreateProcedureStmt        "CREATE PROCEDURE statement"
+	AddQueryWatchStmt       "CREATE QUERY WATCH statement"
```

Comment:
I see a lot of function names are still using `Create`, let's keep `CreateQueryWatchStmt` or change all places.